pkg/R/DataFrame.R

```diff
@@ -768,6 +798,20 @@ setMethod("select", signature(x = "DataFrame", col = "Column"),
             dataFrame(sdf)
           })
 
+setMethod("select",
+          signature(x = "DataFrame", col = "list"),
```

Comment:
What's the purpose of this API? I did not find one in Scala/Python to match this.